src/main/webapp/populateData.js

```diff
@@ -154,3 +160,76 @@ function populateCalendar() {
         }
       });
 }
+
+/**
+ * Function to test getting taskLists and adding a new taskList
+ * Will 1) request a new taskList be made with a default name (the current time)
+ * and 2) get the new list of taskLists and log them in the console.
+ */
+function postAndGetTaskList() {
+  const sampleTitle =
+      getDateInLocalTimeZone().getTime().toString();
+
+  postNewTaskList(sampleTitle)
+      .then(() => {
+        getTaskListsAndTasks()
+            .then(() => {
+              console.log(taskLists);
+              console.log(tasks);
+            });
+      });
+}
+
+/**
+ * Update the tasks and taskLists lists.
+ *
+ * @return {Promise<any>} A promise that is resolved once the tasks and
```

Comment:
Where? (same below)

Reply:
`fetch(url)` is a promise - I returned the entire fetch block, which is a promise

Reply:
ACK